Samples.Swift/Samples.Swift/AppDelegate.swift

```diff
@@ -1,25 +1,42 @@
 //
 //  AppDelegate.swift
-//  PayPalCheckout-Samples-iOS-Swift
+//  paypalcheckout-iOS-swift-sample
 //
-//  Created by Haider Khan on 5/23/20.
-//  Copyright © 2020 PayPal. All rights reserved.
+//  Created by Jax DesMarais-Leder on 3/4/21.
+// Copyright © 2021 PayPal. All rights reserved.
 //
 
 import UIKit
+import PayPalCheckout
 
-@UIApplicationMain
+@main
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
-
+  var window : UIWindow?
 
   func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-    // Override point for customization after application launch.
+    if #available(iOS 13, *) {
+      setConfig()
+    }
+    else {
+      setConfig()
+      self.window = UIWindow()
+      let viewController = CheckoutViewController()
+      window?.rootViewController = viewController
+      window?.makeKeyAndVisible()
+    }
     return true
   }
 
-  // MARK: UISceneSession Lifecycle
+  func setConfig() {
+    let config: Config = {
+      let config = Config(clientID: PayPal.clientId, returnUrl: PayPal.returnUrl, environment: .sandbox)
```

Comment:
Can you give a new line for each of the params?

Reply:
Also no need for closure returning a config object i don't think. You can just create it directly.